src/logging/ConsoleLogger.ts

```diff
@@ -0,0 +1,43 @@
+import { Logger, Context } from "./Logger";
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
?

Reply:
see: https://github.com/LBHackney-IT/housing-needs-evidence-store/pull/12#discussion_r452186503